### PR TITLE
Serverless architecture 

### PR DESCRIPTION
**Pull request by plantysnake**

### CODE REVIEW

File: src/handler.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+const fs = require("fs")
+const { promisify } = require('util')
+const readFile = promisify(fs.readFile)
+
+module.exports.getTableData = async (event) => {
+  var dataStr = event.queryStringParameters.data || "data"
+  return readFile("src/data/" + dataStr + ".json", {}).then((data) => {
```
Comment (marina-igorevna-mishur) [inline src/handler.js]: why is data in src?

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "eslint": "eslint src/js/**/*.{js,jsx}",
+    "debug": "node ./node_modules/serverless/bin/serverless offline ----httpPort 3001",
     "start": "node src/js/server.js"
```
Comment (marina-igorevna-mishur) [inline package.json]: please update start script

Comment (plantysnake) [reply package.json]: Done

File: src/handler.js (hunk 1 above)
Comment (plantysnake) [reply src/handler.js]: Moved

File: src/getTrial.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const fs = require("fs").promises
+
+module.exports.getTrial = async (event) => {
+    const id = event.pathParameters.trialId
+    const trial = JSON.parse(await fs.readFile("data/data.json")).trials.filter(p => p.id == id)[0]
```
Comment (marina-igorevna-mishur) [inline src/getTrial.js]: `filter()[0]` = `find()`

**Comment by marina-igorevna-mishur** [comment]
let's close this one

**Pull request closed without merging**